staging/src/k8s.io/apimachinery/pkg/api/resource/quantity_proto.go

```diff
@@ -77,14 +88,7 @@ func (m *Quantity) Size() (n int) {
 }
 
 func sovGenerated(x uint64) (n int) {
-	for {
-		n++
-		x >>= 7
-		if x == 0 {
-			break
-		}
-	}
-	return n
+	return (bits.Len64(x|1) + 6) / 7
```

Comment:
Did this get changed in the other generated methods?  I didn't see sovGenerated changing but maybe I just missed it?

Reply:
Nm, I see it now.